File: UnitTestProject1/AppManager.cs
using System;
using System.Collections.Generic;
using System.Text;
using OpenQA.Selenium;
using OpenQA.Selenium.Firefox;

namespace UnitTestProject1
{
    public class AppManager
    {
        private IWebDriver driver;
        private IJavaScriptExecutor js;
        private StringBuilder verificationErrors;
        private string baseURL;

        private NavigationHelper navigation;
        private LoginHelper auth;
        private TodoHelper todo;

        public IDictionary<string, object> Vars { get; set; }

        public AppManager()
        {
            driver = new FirefoxDriver();
            js = (IJavaScriptExecutor)driver;
            Vars = new Dictionary<string, object>();
            driver.Manage().Window.Size = new System.Drawing.Size(1252, 694);

            baseURL = "https://www.todoist.com/";
            verificationErrors = new StringBuilder();

            todo = new TodoHelper(this);
            auth = new LoginHelper(this);
            navigation = new NavigationHelper(this, baseURL);
        }

        public IWebDriver Driver
        {
            get
            {
                return driver;
            }
        }

        public IJavaScriptExecutor Js
        {
            get
            {
                return js;
            }
        }

        public NavigationHelper Navigation
        {
            get
            {
                return navigation;
            }
        }

        public LoginHelper Auth
        {
            get
            {
                return auth;
            }
        }

        public TodoHelper Todo
        {
            get
            {
                return todo;
            }
        }

        public void Stop()
        {
            driver.Quit();
        }
    }
}

File: src/test/java/ru/itis/todoist/AppManager.java
package ru.itis.todoist;

import org.openqa.selenium.Dimension;
import org.openqa.selenium.JavascriptExecutor;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import ru.itis.todoist.helpers.LoginHelper;
import ru.itis.todoist.helpers.NavigationHelper;
import ru.itis.todoist.helpers.TodoHelper;

import java.util.HashMap;
import java.util.Map;

public class AppManager {
    private final WebDriver driver;
    private final JavascriptExecutor js;
    private final NavigationHelper navigation;
    private final LoginHelper auth;
    private final TodoHelper todo;
    private final Map<String, Object> vars = new HashMap<>();

    public AppManager() {
        driver = new FirefoxDriver();
        js = (JavascriptExecutor) driver;
        driver.manage().window().setSize(new Dimension(1252, 694));

        String baseUrl = "https://www.todoist.com/";
        todo = new TodoHelper(this);
        auth = new LoginHelper(this);
        navigation = new NavigationHelper(this, baseUrl);
    }

    public WebDriver getDriver() {
        return driver;
    }

    public JavascriptExecutor getJs() {
        return js;
    }

    public NavigationHelper getNavigation() {
        return navigation;
    }

    public LoginHelper getAuth() {
        return auth;
    }

    public TodoHelper getTodo() {
        return todo;
    }

    public Map<String, Object> getVars() {
        return vars;
    }

    public void stop() {
        driver.quit();
    }
}
